lib/plug/router.ex

```diff
@@ -85,6 +85,33 @@ defmodule Plug.Router do
   Check `match/3` for more information on how route compilation
   works and a list of supported options.
 
+  ## Parameter Parsing
+
+  Handling request data can be done through the
+  [`Plug.Parsers`](https://hexdocs.pm/plug/Plug.Parsers.html#content) module. It
+  provides support for parsing url encoded, form-data, and json data as well as
+  providing a behaviour that all other Parsers should adopt.
+
+  Here is an example of how it can be used to parse a JSON POST Body:
+
+      defmodule AppRouter do
+        use Plug.Router
+
+        plug Plug.Parsers, parsers: [:json],
```

Comment:
Well, it matters in some cases (likely not this one), because there are some actions you may want to execute only if there is a valid route, such as fetching the session or authentication.